README.md

```diff
@@ -21,13 +21,21 @@ A thin layer of TiSpark. Most of the logic is inside tikv-java-client library.
 https://github.com/pingcap/tikv-client-lib-java
 
 
-Uses as below
+## Usage
+### For spark-shell
+1. Download or compile tispark [here](https://github.com/pingcap/tispark).
+2. ```cd``` to your spark configuration directory and add this line in your ```spark-default.conf```
 ```
-./bin/spark-shell --jars /wherever-it-is/tispark-0.1.0-SNAPSHOT-jar-with-dependencies.jar
+spark.tispark.pd.addresses 127.0.0.1:2379
```

Comment:
please specify that here should put your pd address instead of just localhost.

Reply:
done